src/main/java/com/google/cloud/tools/appengine/cloudsdk/CloudSdk.java

```diff
@@ -377,11 +376,28 @@ public Path getJarPath(String jarName) {
   }
 
   /**
-   * Checks whether the Cloud SDK Path with is valid.
+   * Checks whether the Cloud SDK path and version are valid.
    *
-   * @throws CloudSdkNotFoundException when the Cloud SDK is not installed where expected
+   * @throws CloudSdkNotFoundException when an up-to-date Cloud SDK is not installed where expected
    */
-  public void validateCloudSdk() throws CloudSdkNotFoundException {
+  public void validateCloudSdk() {
+    validateCloudSdkLocation();
```

Comment:
I've seen using throws for a runtime exception be a kind of middle ground between the checked/unchecked exception crowds.  I don't think we need to remove it, and having it there makes it harder to miss and the IDE try/catch generation work better.